.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: ruby
 rvm:
   # SLES15
-  - 2.5.0
+  - 2.5.8
```

Comment:
I suggest to use SCC package search to find all Ruby versions that we need to support and add them here.

Reply:
![Screenshot_20210518_154822](https://user-images.githubusercontent.com/892391/118663256-f8d9f600-b7f0-11eb-9e47-1cafcee73273.png)

Not sure if we should do it like that, if I search in package search for Ruby package we have all versions there
maybe is enough to have latest minor versions only, which PR does?

Reply:
I added the latest minor version of Ruby 2.5+